applications/crossbar/src/modules/provisioner_v5.erl

```diff
@@ -31,14 +32,20 @@
 %%--------------------------------------------------------------------
 -spec put(ne_binary(), wh_json:object()) -> 'ok'.
 put(JObj, AuthToken) ->
+    AccountId = wh_json:get_value(<<"pvt_account_id">>, JObj),
     Data = provision_data(JObj),
-    MACAddress = wh_json:get_value(<<"mac_address">>, JObj),
-    _ = send_req('devices_put'
-                 ,Data
-                 ,AuthToken
-                 ,wh_json:get_value(<<"pvt_account_id">>, JObj)
-                 ,MACAddress),
-    send_req('files_post', AuthToken, MACAddress).
+    case check_data(wh_json:set_value(<<"merge">>, 'false', Data)) of
```

Comment:
It is done by the schema default.
On Jan 27, 2015 5:59 PM, "bitbashing" <notifications@github.com> wrote:

> In applications/crossbar/src/modules/provisioner_v5.erl
> <https://github.com/2600hz/kazoo/pull/732#discussion_r23660803>:
>
> >      Data = provision_data(JObj),
> > -    MACAddress = wh_json:get_value(<<"mac_address">>, JObj),
> > -    _ = send_req('devices_put'
> > -                 ,Data
> > -                 ,AuthToken
> > -                 ,wh_json:get_value(<<"pvt_account_id">>, JObj)
> > -                 ,MACAddress),
> > -    send_req('files_post', AuthToken, MACAddress).
> > +    case check_data(wh_json:set_value(<<"merge">>, 'false', Data)) of
>
> merge should be set to 'true'
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/2600hz/kazoo/pull/732/files#r23660803>.
>